src/ledger.ts

```diff
@@ -369,13 +375,14 @@ export class LedgerBitcoinInteraction extends LedgerInteraction {
    * as well as set the properties of support before calling their run
    * in order to check support before calling the actual interaction run
    */
-  async run() {
+  async run(): Promise<unknown> {
```

Comment:
Unfortunately not because there are other classes that inherit from it and so if it's void then the children classes that do return something will complain

Reply:
There is a way to reference the polymorphic type to perform conditional typing. Is there is a parent class that's actually an abstract class. Is that `LedgerInteraction`? Then you can do something like 
```
type ChildTypes = LedgerBitcoinInteraction | SomethingElse;

abstract class LedgerInteraction<InstanceType extends ChildTypes = ChildTypes> {
  //  and in here you can reference InstanceType
}
```

Reply:
It probably is too heavy for now, at least until everything is typescriptified or refactored. using `unknown` is like using `any`, it breaks typescripts typing and should be avoided.